File-Formats/DocIO/Word-document/Compare Word documents.md

```diff
@@ -0,0 +1,229 @@
+
+The following code example shows how to set author and date for revision while comparing two Word documents.
+
+{% tabs %}
+{% highlight c# tabtitle="C# [Cross-platform]" %}
+
+//Loads the original document.
+using (FileStream originalDocumentStreamPath = new FileStream("Data/OriginalDocument.docx", FileMode.Open, FileAccess.Read))
+{
+    using (WordDocument originalDocument = new WordDocument(originalDocumentStreamPath, FormatType.Docx))
+    {
+        //Loads the revised document.
+        using (FileStream revisedDocumentStreamPath = new FileStream("Data/RevisedDocument.docx", FileMode.Open, FileAccess.Read))
+        {
+            using (WordDocument revisedDocument = new WordDocument(revisedDocumentStreamPath, FormatType.Docx))
+            {
+                // Compare the original and revised Word documents.
+                originalDocument.Compare(revisedDocument,"Nancy Davolio", DateTime.Now.AddDays(-1));
+
+                //Saves the Word document to MemoryStream
+                MemoryStream stream = new MemoryStream();
+                originalDocument.Save(stream, FormatType.Docx);
+            }
+        }                 
+    }                           
+}
+
+
+{% endhighlight %}
+{% highlight c# tabtitle="C# [Windows-specific]" %}
+
+//Loads the original document.
+using (WordDocument originalDocument = new WordDocument("Data/OriginalDocument.docx", FormatType.Docx))
+{
+    //Loads the revised document.
+    using (WordDocument revisedDocument = new WordDocument("Data/RevisedDocument.docx", FormatType.Docx))
+   {
+        /Compare original document and revised documents.
+        originalDocument.Compare(revisedDocument,"Nancy Davolio", DateTime.Now.AddDays(-1));
+        //Save the Word document.
+        originalDocument.Save("Result.docx");          
+    }
+}
+
+{% endhighlight %}
+{% highlight vb.net tabtitle="VB.NET [Windows-specific]" %}
+
+' Open the original Word document.
+Using originalDocument As New WordDocument(originalFilePath, FormatType.Docx)
+    ' Open the revised Word document.
+    Using revisedDocument As New WordDocument(revisedFilePath, FormatType.Docx)
+        ' Compare original document with revised document.
+        originalDocument.Compare(revisedDocument, "Nancy Davolio", DateTime.Now.AddDays(-1))
+        ' Save the Word document.
+        originalDocument.Save(resultFilePath)
+    End Using
+End Using
+
+{% endhighlight %}
+{% endtabs %}
+
+![Compare Word documents in C#](WorkingwithWordDocument_images/Compare-Word-documents.png)
+
+## Comparison options
+
+You can customize the Word comparison using our ComparisonOptions in DocIO.
+
+### Ignore format changes
+
+In the .NET Word library (DocIO), document comparison includes formatting changes by default. However, you can configure DocIO to ignore formatting differences using DetectFormatChanges API to concentrate solely on content modifications.
```

Comment:
In the .NET Word library (DocIO), document comparison includes formatting changes by default. However, you can configure DocIO to ignore formatting differences using the DetectFormatChanges API to concentrate solely on content modifications.